save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/AgentStatus.kt

```diff
@@ -42,8 +42,19 @@ class AgentStatus(
  * @property containerId id of the agent's container
  * @property time
  */
-class AgentStatusDto(
+data class AgentStatusDto(
     val time: LocalDateTime,
     val state: AgentState,
     val containerId: String,
 )
+
+/**
+ * Statuses of a group of agents for a single Execution
+ *
+ * @property executionId id of Execution
+ * @property agentStatuses list of [AgentStatusDto]s
+ */
+data class AgentStatusesForExecution(
```

Comment:
May be just add `executionId` to `AgentStatusDto` to avoid new class?

Reply:
I think this class can more clearly indicate, that all agents have the same `executionId` here, and also removes a little of data duplication.